### ISSUE TITLE
Support page needs a `<main>` landmark

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4042]
## Area or Component
simplereport.gov/support

## Quick Summary
The support home page doesn't have a `<main>` section, making it difficult to navigate between content.

## Accessibility Violation
Document should have one main landmark: https://dequeuniversity.com/rules/axe/4.4/landmark-one-main?application=AxeChrome

## Action Requested
Add a `<main>` landmark to the support page.

This will resolve several related issues around "All page content should be contained by landmarks".

## Original Deque Issue
https://docs.google.com/spreadsheets/d/16udNpwdJ8sltAULnmSIGgkWVQsfuFvcoymP0-s_QwfE/edit#gid=1832428693&range=16:16
